models/namespace.js

```diff
@@ -51,5 +83,46 @@ export default {
 
   projectNameSort() {
     return this.project?.nameSort || '';
+  },
+
+  istioInstalled() {
+    const schema = this.$rootGetters['cluster/schemaFor'](ISTIO.GATEWAY);
+
+    return !!schema;
+  },
+
+  injectionEnabled() {
+    return this.labels[ISTIO_LABELS.AUTO_INJECTION] === 'enabled';
+  },
+
+  enableAutoInjection() {
+    return (namespaces = this) => {
+      this.toggleAutoInjection(namespaces);
+    };
+  },
+
+  disableAutoInjection() {
+    return (namespaces = this) => {
+      this.toggleAutoInjection(namespaces);
+    };
+  },
```

Comment:
These both say enable/disable but are actually toggling (and identical).  Pass down an explicit enable boolean so that calling `enableAutoInjection` on a namespace that already has it on has no effect instead of unexpectedly turning it off.